integration/entropy/entropy_time.bats

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source /etc/os-release || source /usr/lib/os-release
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	clean_env
+	# Check that processes are not running
+	run check_processes
+	echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "entropy for /dev/random" {
+	output_file=$(mktemp)
+	block_size="4b"
+	entropy_time="10"
```

Comment:
should we add a comment of why this is a good value?
also, can the variable name be changed to something like `expected_time` ?